### ISSUE TITLE
Wrong base64 encoding on multybyte utf-8 characters

### ISSUE THREAD

**Issue by microstudi** [vocdoni/vocdoni-sdk#90]
While working on this SDK, we've found that if you use multibyte UTF8 chars in a `UnpublishedElection` object (either in the description or in any of the questions), the result is not coded properly.

I believe that the cause is this line, using "binary" as encoding.
https://github.com/vocdoni/vocdoni-sdk/blob/93dfae5a4bdac8641a8fb882669d448843b490b3/src/core/account.ts#L48

According to https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings, "binary" is an alias for "latin1".

I believe this should be using "utf8" instead. Otherwise we cannot use it for languages like Japanese.

**Comment by andreslucena** [comment]
This can be quickly checked on any of the examples, for instance:

```diff
diff --git a/examples/esm/src/index.mjs b/examples/esm/src/index.mjs
index b526c07..d0ea2f4 100644
--- a/examples/esm/src/index.mjs
+++ b/examples/esm/src/index.mjs
@@ -9,7 +9,7 @@ const createElection = (census, electionType = null) => {
   endDate.setHours(endDate.getHours() + 10);
 
   const election = Election.from({
-    title: 'Election title',
+    title: 'こんにちは世界',
     description: 'Election description',
     header: 'https://source.unsplash.com/random',
     streamUri: 'https://source.unsplash.com/random',
```

**Comment by microstudi** [comment]
Thanks, I've incorporated in the PR!

**Comment by marcvelmer** [comment]
Fixed in #116
